application_integration/main.tf

```diff
@@ -0,0 +1,39 @@
+
+data "google_project" "test_project" {
+}
+
+# [START application_integration_edit_region]
+
+resource "google_kms_key_ring" "keyring" {
+  name     = "my-keyring"
+  location = "us-east1"
+}
+
+resource "google_kms_crypto_key" "cryptokey" {
+  name            = "crypto-key-example"
```

Comment:
Kindly check other KMS examples in this repo for reference.

KMS ring name needs a random suffix.